x-pack/plugins/ml/public/components/field_data_card/_field_data_card.scss

```diff
@@ -68,6 +68,13 @@
     display: inline-block;
   }
 
+  .stat.min.value, .stat.max.value, .stat.median.value {
```

Comment:
`@include euiTextTruncate` might give you an easier way to manage this. It'll also work cross browser.

https://github.com/elastic/eui/blob/master/src/global_styling/mixins/_typography.scss#L108-L125